<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PredictionDialog</class>
 <widget class="QDialog" name="PredictionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stroke Risk Prediction Questionnaire</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Please fill in your health indicators:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>660</width>
        <height>525</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsAndLabels</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::WrapLongRows</enum>
       </property>
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="genderLabel">
         <property name="text">
          <string>Gender:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="genderComboBox">
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ageLabel">
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="ageSpinBox">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="hypertensionLabel">
         <property name="text">
          <string>Hypertension (High Blood Pressure):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="hypertensionComboBox">
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="heartDiseaseLabel">
         <property name="text">
          <string>Heart Disease:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="heartDiseaseComboBox">
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="everMarriedLabel">
         <property name="text">
          <string>Ever Married:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="everMarriedComboBox">
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="workTypeLabel">
         <property name="text">
          <string>Work Type:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="workTypeComboBox">
         <item>
          <property name="text">
           <string>Private</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Self-employed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Govt_job</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>children</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Never_worked</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="residenceTypeLabel">
         <property name="text">
          <string>Residence Type:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="residenceTypeComboBox">
         <item>
          <property name="text">
           <string>Urban</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rural</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="avgGlucoseLabel">
         <property name="text">
          <string>Avg Glucose Level:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="avgGlucoseLineEdit">
         <property name="placeholderText">
          <string>e.g., 90.0</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="bmiLabel">
         <property name="text">
          <string>BMI:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="bmiLineEdit">
         <property name="placeholderText">
          <string>e.g., 25.5</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="smokingStatusLabel">
         <property name="text">
          <string>Smoking Status:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="smokingStatusComboBox">
         <item>
          <property name="text">
           <string>formerly smoked</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>never smoked</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>smokes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="predictButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Predict Stroke Risk</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Prediction: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
